### PR TITLE
insights: update process page

### PR DESCRIPTION
**Pull request by coury-clark**
First pass at some basic documentation for the new planning board, as well as cleaning up incorrect or outdated information

### CODE REVIEW

File: content/departments/engineering/teams/code-insights/processes.md [hunk 1]
```diff
@@ -16,41 +16,31 @@ Attendees are encouraged to add comments on talking points asynchronously before
 
 ## Planning and prioritization
 
-We plan and track our day-to-day work on our [GitHub board](https://github.com/orgs/sourcegraph/projects/200/views/1).
-The project is separated into multiple views: **Current iteration**, **Next iteration** (both grouped by issue status) and **All iterations** (grouped by milestone).
+We plan and track our day-to-day work on our [GitHub board v2](https://github.com/orgs/sourcegraph/projects/291/views/1). This board has the philosophy that is tries to reflect reality as best as possible. Flexibility is a key component of how to use this board and it should support making decisions that align the team and allow us to move forward with our work.
```
Comment (leonore) [inline content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
We plan and track our day-to-day work on our [GitHub board](https://github.com/orgs/sourcegraph/projects/291/views/1). This board follows the philosophy of trying to reflect reality as best as possible. Flexibility is a key component of how to use this board and it should support making decisions that align the team and allow us to move forward with our work.
```
I don't think the `v2` is required here, does the reader need to know this is our second pass at a board?

File: content/departments/engineering/teams/code-insights/processes.md [hunk 2]
```diff
@@ -16,41 +16,31 @@ Attendees are encouraged to add comments on talking points asynchronously before
...
 
-We use an intentional list of statuses on our project board:
+The board is organized around a priority structure where p0 is the most important column and p2 is the least important, followed by the backlog. Issues in the p0-p2 priority pipeline are issues that are planned for work. The backlog is for issues to go that are not currently prioritized but are still loosely on the product roadmap.
```
Comment (leonore) [inline content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
The board is organized around a priority structure where P0 is the most important column and P2 is the least important, followed by the backlog. Issues in the P0-P2 priority pipeline are issues that are planned for work. The backlog is for issues to go that are not currently prioritized but are still loosely on the product roadmap.
```

File: content/departments/engineering/teams/code-insights/processes.md [hunk 3]
```diff
@@ -16,41 +16,31 @@ Attendees are encouraged to add comments on talking points asynchronously before
...
-- **Backlog**: We are keeping this issue open for reference, as a placeholder, and for future prioritization, but it's not a current priority.
-- **On deck**: These issues are not currently proposed for the upcoming iteration but are a maintained, priority ordering of likely next issues.
-- **Proposed for iteration**: These are a select group of issues proposed for a specific iteration. They must have an iteration milestone attached.
-- **Todo**: This is the small set of issues that are planned for our current iteration.
-- **In Progress**: Issues currently being worked on in the iteration.
-- **In Review**: Issues that have PRs in review.
-- **Done**: All closed issues, including "won't do" issues.
+Issues can freely move between priority columns as required. For example, if something important shows up in triage it will likely knock other things out of upcoming priority.
+
+Ideally every ticket that moves into p0 is work that someone is actually going to do (assigned to someone) and has a rough sense of size. It is less important than issues cleanly "fit" in an iteration and more important that everyone clearly understands the priority of our work and what they are working on.
```
Comment (leonore) [inline content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
Ideally every ticket that moves into P0 is work that someone is actually going to do (assigned to someone) and has a size estimation. It is less important than issues cleanly "fit" in an iteration and more important that everyone clearly understands the priority of our work and what they are working on.
```

File: content/departments/engineering/teams/code-insights/processes.md [hunk 4]
```diff
@@ -16,41 +16,31 @@ Attendees are encouraged to add comments on talking points asynchronously before
...
+- Incoming tickets are added to the [GitHub project](https://github.com/orgs/sourcegraph/projects/291/views/1) automatically when they are labelled <span class="badge" style="background: var(--bs-teal)">team/code-insights</span> and will be visible in the triage column.
 
 - **While an iteration is ongoing, we plan the next iteration**. This is a mostly asynchronous process.
 
-  - Engineers, designer, PM and EM can **_propose_ issues to be worked on** _before_ the mid-iteration [Thursday iteration planning](#fortnightly-iteration-planning) by adding them to the next iteration milestone and setting "Status" to **Proposed**. This is done considering our [higher-level goals and roadmap](../../../../strategy-goals/strategy/code-insights/index.md).
+  - Anyone can add issues to the triage column before the [Thursday iteration planning](#fortnightly-iteration-planning). Issues will be triaged and sorted into priority columns. Issues that are going to be worked on immediately will receive an assignee and a rough estimate before moving into p0.
```
Comment (leonore) [inline content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
  - Anyone can add issues to the triage column before the Thursday iteration planning. Issues will be triaged and sorted into priority columns. Issues that are going to be worked on immediately will receive an assignee and a rough estimate before moving into P0.
```

File: content/departments/engineering/teams/code-insights/processes.md [hunk 5]
```diff
@@ -126,17 +107,13 @@ While there is no "maximum" number, once we get higher than a 13 in size it will
 
 ### Fortnightly iteration planning
 
-Every two weeks, usually on the Thursday before the next iteration begins, we hold an iteration planning. This is a short sync meeting to answer questions or align on the priority of issues planned for the next iteration. This includes: questions about priority, clarity, or edge cases of specific features for the next iteration; refining plans for the following Monday iteration start; surfacing any missing GitHub issues (for known items) ahead of the Monday kickoff so they can be made before Monday.
+Every two weeks, usually on the Thursday before the next iteration begins, we hold an iteration planning. This is a sync meeting to triage new issues for priority, review the priority of issues that are still in progress, and review the priority of issues already in the upcoming pipeline.
```
Comment (leonore) [inline content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
Every two weeks, usually on the Thursday before the next iteration begins, we hold an iteration planning. This is a sync meeting to triage new issues according to priority, review the priority of issues that are still in progress, and review the priority of issues already in the upcoming pipeline.
```

**Review by leonore — COMMENTED** [review]
thanks for taking the first go at this, I was thinking it might be time to trim down this page! just some suggestions for wording, overall lgtm but won't approve just yet to give time for anyone to voice any opposition if desired

File: content/departments/engineering/teams/code-insights/processes.md (hunk 4 above)
Comment (felixfbecker) [inline content/departments/engineering/teams/code-insights/processes.md]: I find it a bit confusing to say that anyone can add issues to the column before Thursday if all issues get added automatically, i.e. should always already be there (unless they're in a different column already). Is the suggestion here for people to _move_ issues from a different column to "Triage" for reevaluation?

Comment (felixfbecker) [reply content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
  - Anyone can add issues to the triage column before the [Thursday iteration planning](#fortnightly-iteration-planning). Issues will be triaged and sorted into priority columns. Issues that are going to be worked on immediately will receive an assignee and a rough estimate range before moving into p0.
```

**Review by felixfbecker — APPROVED** [review]
Thank you for documenting! 💯

File: content/departments/engineering/teams/code-insights/processes.md (hunk 4 above)
Comment (coury-clark) [reply content/departments/engineering/teams/code-insights/processes.md]: I'm not entirely sure I understand that interpretation. Anyone can add issues to the triage column either automatically or manually. Once they are in the triage column they will be placed into priority columns. I'm not following any reason someone would read this and think they need to move issues back.

Comment (coury-clark) [inline content/departments/engineering/teams/code-insights/processes.md]:
```suggestion
  - Anyone can add issues to the triage column before the [Thursday iteration planning](#fortnightly-iteration-planning). Issues will be triaged and sorted into priority columns. Issues that are going to be worked on immediately will receive an assignee and a rough estimate range before moving into p0.
```

**Pull request merged**